src/components/form/examples/basic-form.tsx

```diff
@@ -1,9 +1,9 @@
 import { ValidationStatus } from '@limetech/lime-elements';
 import { Component, h, State } from '@stencil/core';
-import { schema } from './schema';
+import { schema } from './basic-schema';
```

Comment:
I renamed a bunch of these files, so it would be easier to know which belong together, and which example they provide.